raiden/utils/cli.py

```diff
@@ -371,10 +372,14 @@ def fixed_gas_price_strategy(_web3, _transaction_params):
                 # we assume it is already the correct gas-price-strategies function
                 return value
             gas_price_string = super().convert(value, param, ctx)
-            if gas_price_string == "fast":
+            if gas_price_string == "normal":
+                return fast_gas_price_strategy
+            elif gas_price_string == "fast":
                 return faster_gas_price_strategy
+            elif gas_price_string == "rpc":
+                return rpc_gas_price_strategy
             else:
-                return fast_gas_price_strategy
+                raise ConfigurationError(f"Unknown gas price (strategy): {gas_price_string}")
```

Comment:
Using a dictionary would make this more maintainable, and easier to read:

```python
gas_strategies = {
    "normal": fast_gas_price_strategy,
    "fast": faster_gas_price_strategy,
    "rpc": rpc_gas_price_strategy
}

if gas_price_string not in gas_strategies:
    raise ConfigurationError(f"Unknown gas price (strategy): {gas_price_string}")

return gas_strategies[gas_price_string]
```

Reply:
Good idea! I'll take that.

Reply:
I'll just amend the commit as it is a single one and I don't see much sense to push a fixup commit here that then needs to be rebased. I think the commit is small enough that it is easy for everyone to review the changes due to this conversation.

Reply:
✔️